bundles/org.openhab.core.addon.marketplace/src/main/java/org/openhab/core/addon/marketplace/internal/community/CommunityMarketplaceAddonService.java

```diff
@@ -191,7 +192,7 @@ public void refreshSource() {
                     .map(t -> convertTopicItemToAddon(t, users)).collect(Collectors.toList());
         } catch (Exception e) {
             logger.error("Unable to retrieve marketplace add-ons", e);
-            return new ArrayList<Addon>();
+            return new ArrayList<>();
```

Comment:
```suggestion
            return List.of();
```

Less code and it returns a constant which reduces memory consumption and garbage collection.

Reply:
Done.